pkgs/by-name/zo/zoom-us/update.sh

```diff
@@ -4,7 +4,7 @@
 set -eu -o pipefail
 
 scriptDir=$(cd "${BASH_SOURCE[0]%/*}" && pwd)
-nixpkgs=$(realpath "$scriptDir"/../../../../..)
+nixpkgs=$(realpath "$scriptDir"/../../../..)
```

Comment:
Well, the CI can't see this (since it is not a Nix file), but it violates the "hermetic principle" from RFC 140, in which paths outside the directory should not be accessible.

Reply:
I think as an upgrade script, it's fine. Perhaps some future person (yourself even) will cause this script to be made better.